features/0728-device-binding-attachments/README.md

```diff
@@ -0,0 +1,261 @@
+# Aries RFC 0728 : Device Binding Attachments
+- Authors: [Paul Bastian](mailto:paul.bastian@bdr.de), [Sebastian Bickerle](mailto:sebastian.bickerle@main-incubator.com)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2022-04-07
+- Status Note: This is an initial draft 
+- Start Date: 2022-01-01
+- Tags: [feature](/tags.md#feature)
+
+## Summary
+
+Extends existing present-proof protocols to allow proofing the control of a hardware bound key embedded within a verfiable credential.
+
+## Motivation
+
+To enable use-cases which require a high level of assurance a verifier must reach a high degree of confidence that a verifiable credential (VC) can only be used by the person it was issued for. One way to enforce this requirement is that the issuer additionally binds the VC to a hardware bound public key and therefore binding the credential to the device, as discussed in the [DIF Wallet Security WG](https://github.com/decentralized-identity/wallet-security). The issaunce process, including the attestation of the wallet and the hardware bound key is off-scope for this Aries RFC. A valid presentation of the VC then requires an additional challenge which proofs that the presenter is in control of the corresponding private key.
+
+Since the proof of control must be part of legitimate presentation it makes sense to extend all current `present-proof` protocols.
+
+Note: The focus so far has been on AnonCreds, we will also look into device binding of W3C VC, however this is currently lacking in the examples.
+
```

Comment:
A warning here about inappropriate use of this tech?

Reply:
done